### PR TITLE
Make sure people understand the filter DCA option is an array of arrays

### PR DESCRIPTION
**Pull request by Toflar**

### CODE REVIEW

**Review by aschempp — APPROVED** [review]

**Comment by fritzmg** [comment]
Btw. should we use the `[]` notation instead everywhere (to save some space, especially in those tables)?

**Comment by Toflar** [comment]
Jup, we should but I figured not in this PR :)

**Comment by fritzmg** [comment]
K, I have assigned it to me: #122

File: docs/dev/reference/dca/list.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ $GLOBALS['TL_DCA']['tl_example']['list']['sorting'] = [
...
-| filter                | Query filter (`array`)           | Allows you to add custom filters as arrays, e.g. `array('status=?', 'active')`.                                                                                                                                                                                                                                                                                                                        |
+| filter                | Query filter (`array`)           | Allows you to add custom filters as arrays, e.g. `array(('status=?', 'active'), array('usages>?', 0))`.                                                                                                                                                                                                                                                                                                                        |
```
Comment (fiedsch) [inline docs/dev/reference/dca/list.md:33]: the example misses an `array` it should be 
```
array(array('status=?', 'active'), array('usages>?', 0))
```
or 
```
[['status=?', 'active'], ['usages>?', 0]]
```
as already discussed.

**Pull request merged**